keps/sig-windows/1981-windows-privileged-container-support/README.md

```diff
@@ -628,24 +668,26 @@ in back-to-back releases.
 -->
 
 Alpha
-- Version of containerd: 	v1.5
-- Version of Kubernetes: 	Target 1.20 or 1.21  
-- Version of OS support: 	1809/Windows 2019 LTSC and 2004
-- Alpha Feature Gate for passing privilege flag to CRI
+
+- Version of containerd: v1.5
+- Version of Kubernetes: Target  1.21
+- Version of OS support: 1809/Windows 2019 LTSC and 2004
+- Alpha Feature Gate for passing privileged flag **or** annotaions to CRI calls.
 
 Beta
-- Go through PSP Linux test (e2e: validation & conformance) and make them relevant for Windows (which apply, which dont and where we need to write new tests). 
+
+- Go through PSP Linux test (e2e: validation & conformance) and make them relevant for Windows (which apply, which dont and where we need to write new tests).
 - Provide guidance similar to Pod Security Standards for Windows privileged containers
-- Containerd: 			v1.5
-- Kubernetes 			Target 1.21 or 1.22
-- OS support: 			1809/Windows 2019 LTSC and 2004
+- Containerd: v1.5
+- Kubernetes Target 1.22 or later
+- OS support: 1809/Windows 2019 LTSC and 2004
```

Comment:
what version of Windows SAC? cc @jeremyje

Reply:
Any and all versions of Windows that support k8s + containerd would support this.
Will update the enhancement.